src/main/webapp/app/course/manage/detail/course-detail.component.html

```diff
@@ -110,6 +114,14 @@ <h2><span jhiTranslate="artemisApp.course.detail.title">Course Details:</span></
                     </dd>
                 </div>
             </ng-template>
+            <dt><span jhiTranslate="artemisApp.course.maxPoints.title">Max. Number of Points</span></dt>
+            <dd>
+                <span>{{ course.maxPoints }}</span>
+            </dd>
+            <dt><span jhiTranslate="artemisApp.course.accuracyOfScores">Amount of decimal places used for calculating the scores</span></dt>
```

Comment:
```suggestion
            <dt><span jhiTranslate="artemisApp.course.accuracyOfScores">Number of decimal places used for calculating the scores</span></dt>
```
The number of decimal places is countable. Can you please also update the existing translation string?